template/.yamllint.yaml

```diff
@@ -8,3 +8,5 @@ rules:
   line-length: disable
   truthy:
     check-keys: false
+  comments:
+    min-spaces-from-content: 1  # Needed due to https://github.com/adrienverge/yamllint/issues/443
```

Comment:
This needs to be done in the main yamllint file as well, no?

Reply:
The yamls in this repo are in general not linted duo to them being called .yaml.j2 - but we could do that as well, yes.

Reply:
I'll approve and leave it up to you whether you want to change it or not